tests/cases/compiler/isolatedModulesReExportAlias.ts

```diff
@@ -0,0 +1,25 @@
+// @isolatedModules: true
+// @noEmit: true
+// @noTypesAndSymbols: true
+
+// @Filename: /events.d.ts
+declare module "events" {
+  interface EventEmitterOptions {
+    captureRejections?: boolean;
+  }
+  class EventEmitter {
+    constructor(options?: EventEmitterOptions);
+  }
+  export = EventEmitter;
+}
+declare module "node:events" {
+  import events = require("events");
+  export = events;
+}
+
+// @Filename: /moreEvents.ts
+import events = require("events");
+export = events;
+
+// @Filename: /boo.ts
+// Bad test runner (ignores stuff when last file contains the string r-e-q-u-i-r-e)
```

Comment:
I am definitely curious about this one... 😄

Reply:
The absolute worst https://github.com/microsoft/TypeScript/blob/25a4f9e53595fe944ada7cc747ec393ce0bc2c6e/src/testRunner/compilerRunner.ts#L238-L242